scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -597,6 +597,17 @@ def add_parse_arguments(parser = None):
         help="Use the list of test scenarios under quarantine and run them"
              "to verify their current status.")
 
+    parser.add_argument(
+        "--scripting-list",
+        action="append",
+        metavar="FILENAME",
+        help="The feature is exclusively compatible with hardware-based "
+         "testing. It requires loading a list of test scenarios accompanied "
+         "by a custom script list. The entries within this list must align "
+         "with the names of the test scenarios as specified in the associated "
```

Comment:
```suggestion
         "by a custom script list. The entries within this list must align "
         "with the names of the test scenarios as specified in the associated "
```
`scenarios` is not a mandatory attribute in the scripting schema, so if it is missed, then is it possible to apply scripts for any scenario or a platform ?

Reply:
changed to be mandatory in scripting-schema.yaml